src/repository/base.repository.js

```diff
@@ -63,6 +63,17 @@ class BaseRepository {
     return model.update(fields, { where: options });
   }
 
+  /**
+   * @static
+   * @param {object} data - object of containing the relevant data
+   * @param {object} options - column options
+   * @returns {object} - returns an database object
+   * @memberof BaseRepository
+   */
+  static async updateField(data, options) {
```

Comment:
There is already an update method in the `baserepository`. please check line 62 [here](https://github.com/andela/ah-kifaru-backend/blob/develop/src/repository/base.repository.js)